### DISCUSSION TITLE
Too many fields. Maximum number of fields is 1000.

### DISCUSSION THREAD

**Discussion by haraleks** [aminalaee/sqladmin#813] [General]
if you add more than 1000 values to the form, an error of 400 occurs (Too many fields. Maximum number of fields is 1000.)

How can this be resolved?

<img width="1426" alt="Снимок экрана 2024-09-06 в 16 14 10" src="https://github.com/user-attachments/assets/6d26f682-10e1-4e72-85c6-ca3ac1c28584">
<img width="216" alt="Снимок экрана 2024-09-06 в 16 14 17" src="https://github.com/user-attachments/assets/c93652f9-d941-4036-9f98-0c2709ac7ace">
<img width="483" alt="Снимок экрана 2024-09-06 в 16 14 22" src="https://github.com/user-attachments/assets/3e73f79f-e283-466c-9405-85e065c3c5b9">

**Comment by aminalaee** [comment]
Please post a minimal code to see what you are doing. It's not clear to me.

**Comment by mmzeynalli** [comment]
Fixed in #1042
